libs/packages/sam-material-extensions/src/lib/table/table.component.ts

```diff
@@ -184,6 +184,7 @@ export class SdsTableComponent implements OnInit, AfterContentInit, AfterViewIni
   }
   private _expansion = false;
 
+  @Input() expansionPosition: string = 'right';
```

Comment:
Might be worth making an enum for this to allow for IDE assistance and allow the compiler to verify that values passed in are valid.